Add Blueprints section to the "How to Contribute" guide (#1137)
kytos/kytos: docs/developer/how_to_contribute.rst

- `Want a Feature?`_
- `Want a Doc Fix?`_
- `Submission Guidelines`_

> - `Blueprint`_

Reviewer: ```suggestion
- `Blueprints`_
```

it is:

- **Major Changes** that you wish to contribute to the project should be
  discussed first in our `Slack channel`_, so that we can better coordinate our 

>   efforts, prevent work duplication, and help you to craft the changes so they 

Reviewer: ```suggestion
  efforts, prevent work duplication, and help you craft the changes so they 
```

it is:

- **Major Changes** that you wish to contribute to the project should be
  discussed first in our `Slack channel <https://join.slack.com/t/kytos/shared_
  invite/enQtMjk0MTM0NjQwOTE1LTE2N2UyNWE2YjNjNzY0MTNiNDNiY2JmNGFlMGQxY2I5Y2IxY

>   jBhMTkwZjZjNDQ4Zjk3ZjExZGFjNGYzMzRjMDM>`_, so that we can better coordinate our 

Reviewer: Please update to use RST reference syntax, to not repeat the URLs.

#.  Create a new PR in GitHub
#.  Respond to any code review feedback

Blueprints
----------

We have a type of document used to describe our architectural choices, called 
‘blueprint’. This document is developed for our developers and records all the
main architectural decisions made in the kytos project. Blueprints are used to
specify the details of any process, resource, and anything else that is
desirable to be implemented, resulting in a major architectural change. You can
check out our blueprints to learn more about our architectural choices in the
`blueprints section`_. Remember that this document is not just for you, but

> mainly for other developers.

Reviewer: Please change "developed for our developers", something like "written for" or "the intended audience are the developers and users of Kytos". It's also important to state that blueprints can be written by anyone, as they are "enhancement proposals" which can be accepted or not.

#.  Create a new PR in GitHub
#.  Respond to any code review feedback

Blueprint
---------

We have a type of document used to describe our architectural choices, called a 
‘blueprint’. This document records all the main architectural decisions made in 
the kytos project, this document is developed for our current developers and 
mainly for our future developers. This document is used to specify the details 
of any process, resource, and anything else that is desirable to be implemented, 
resulting in a major architectural change. You can check out our blueprints to 
learn more about our architectural choices in the `blueprint section`_. 
Remember that this document is not just for you, but mainly for other 

> developers. 

Reviewer: Rephrase "this document..."

> We have a type of document used to describe our architectural choices, called a 
> ‘blueprint’. This document records all the main architectural decisions made in 
> the kytos project, this document is developed for our current developers and 
> mainly for our future developers. This document is used to specify the details 
> of any process, resource, and anything else that is desirable to be implemented, 
> resulting in a major architectural change. You can check out our blueprints to 
> learn more about our architectural choices in the `blueprint section`_. 
> Remember that this document is not just for you, but mainly for other 
> developers. 

Reviewer 2: Maybe it could be something like that:

```suggestion
We have a type of document used to describe our architectural choices, called 
‘blueprint’. This document is developed for our developers and records all the
main architectural decisions made in the kytos project. Blueprints are used to
specify the details of any process, resource, and anything else that is
desirable to be implemented, resulting in a major architectural change. You can
check out our blueprints to learn more about our architectural choices in the
`blueprint section`_. Remember that this document is not just for you, but
mainly for other developers.
```

#.  Create a new PR in GitHub
#.  Respond to any code review feedback

Blueprints
----------

We have a type of document used to describe our architectural choices, called 
‘blueprint’. This document is written for our developers and users and records 
all the main architectural decision proposals that were accepted or not in the 
kytos project. Blueprints are used to specify the details of any process, 
resource, and anything else that is desirable to be implemented, resulting in a 

> major architectural change. You can check out our blueprints to learn more 
> about our architectural choices in the `blueprints section`_. Anyone can make 
> enhancement proposals writing `blueprints`, but first is highly recommended 
> with you know about our past choices by reading in the `blueprints section`_. 
> Remember that this document is not just for you, but mainly for other 
> developers. 

Reviewer: ```suggestion
major architectural change. Anyone can make enhancement proposals 
writing new `blueprints`, but first it is highly recommended that you take
a look at our `blueprints section`_ to read about our past choices. 
Remember that this document is not just for you, but mainly for other 
developers. 
```
